src/account-utils.ts

```diff
@@ -132,34 +63,21 @@ export async function handleAccountContractArtifacts(
         ACCOUNT_CONTRACT_ARTIFACTS_ROOT_PATH
     );
 
-    // Remove old versions from the path
-    if (fs.existsSync(baseArtifactsPath)) {
-        const contents = fs.readdirSync(baseArtifactsPath);
-        contents
-            .filter((content) => content !== ACCOUNT_ARTIFACTS_VERSION)
-            .forEach((content) => {
-                fs.rmSync(path.join(baseArtifactsPath, content), {
-                    recursive: true,
-                    force: true
-                });
-            });
-    }
-
     // Full path to where the artifacts will be saved
     const artifactsTargetPath = path.join(
         baseArtifactsPath,
-        ACCOUNT_ARTIFACTS_VERSION,
+        accountType,
+        artifactsVersion,
         artifactsBase
     );
 
     const jsonArtifact = artifactsName + ".json";
     const abiArtifact = artifactsName + ABI_SUFFIX;
 
     const artifactLocationUrl = GITHUB_ACCOUNT_ARTIFACTS_URL.concat(
```

Comment:
Why are we using Github repo for this? Why don't we just load the artifacts from the directory?

Reply:
Changed downloading from github to copying from the plugin: https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/108/commits/188a6e27ac2a391812d3854ccbc6b28e780d2e3d
Set to unminifed artifacts with debug info: https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/108/commits/c78632c4c863a14df71c73f35129131a6a46b64a